camera_frame_id: usb_cam
ns_tag_detector: usb_cam # proc_image
base_frame_id: base_footprint
odom_frame_id: odom
map_frame_id: map
initial_pose:
- [0.0]
- [0.0]
- [0.0]
tag_pose_id:
# [x, y, theta, id] in /map coordinate system
- [21.6, 15.1, 0.0, 0.0]
- [11.8, 14.5, 0.0, 1.0]
- [21.2, 10.3, 0.0, 2.0]
- [22.7, 10.3, 0.0, 3.0]
- [23.9, 10.3, 0.0, 4.0]
- [25.2, 10.3, 0.0, 5.0]
- [20.95, 15.1, 0.0, 6.0]
- [28.4, 10.3, 0.0, 7.0]
- [30.0, 10.3, 0.0, 8.0]
- [31.6, 10.3, 0.0, 9.0]
- [33.2, 10.3, 0.0, 10.0]
- [26.9, 10.3, 0.0, 11.0]
- [19.55, 10.3, 0.0, 12.0]
- [17.9, 10.3, 0.0, 13.0]
- [16.25, 10.3, 0.0, 14.0]
- [14.6, 10.3, 0.0, 15.0]
- [13.6, 10.3, 0.0, 16.0]
- [24.5, 10.8, 0.0, 17.0]
- [13.4, 12.07, 0.0, 18.0]
- [13.4, 13.65, 0.0, 19.0]
- [17.06, 11.1, 0.0, 20.0]
- [34.8, 10.3, 0.0, 21.0]
- [30.7, 10.7, 0.0, 22.0]
- [32.5, 10.7, 0.0, 23.0]
- [33.9, 10.7, 0.0, 24.0]
- [27.5, 10.7, 0.0, 26.0]
- [26.0, 10.7, 0.0, 27.0]
- [29.1, 10.7, 0.0, 28.0]
- [32.4, 15.1, 0.0, 29.0]
- [24.6, 15.1, 0.0, 30.0]
- [15.6, 17.3, 0.0, 31.0]
- [23.0, 15.1, 0.0, 32.0]
